doc/src/manual/installation.md

```diff
@@ -0,0 +1,83 @@
+# [Installation](@id man-installation)
+
+There are many ways to install Julia. The following sections highlight the recommended method for each of the main supported platforms, and then present alternative ways that might be useful in specialized situations.
+
+Julia switched the default installation recommendation to a solution based on Juliaup in the summer of 2023. If you installed Julia previously with a method that is _not_ based on Juliaup and want to switch your system to an installation that is based on Juliaup, we recommend that you uninstall all previous Julia versions, ensure that you remove anything Julia related from your `PATH` variable and then install Julia with one of the methods described below.
+
+## Windows
+
+On Windows Julia can be installed directly from the Windows store [here](https://www.microsoft.com/store/apps/9NJNWW8PVKMN). One can also install exactly the same version by executing
+
+```
+winget install julia -s msstore
+```
+
+in any shell.
+
+## Mac and Linux
+
+Julia can be installed on Linux or Mac by executing
+
+```
+curl -fsSL https://install.julialang.org | sh
+```
+
+in a shell.
+
+### Command line arguments
+
+One can pass various command line arguments to the Julia installer. The syntax for installer arguments is
+
+```bash
+curl -fsSL https://install.julialang.org | sh -s -- <ARGS>
+```
+
+Here `<ARGS>` should be replaced with one or more of the following arguments:
+- `--yes` (or `-y`): Run the installer in a non-interactive mode. All configuration values use their default or a value supplied as a command line argument.
+- `--default-channel=<NAME>`: Configure the default Juliaup channel. For example `--default-channel lts` would install the `lts` channel and configure it as the default.
+- `--add-to-path=<yes|no>`: Configure whether Julia should be added to the `PATH` environment variable. Valid values are `yes` (default) and `no`.
+- `--background-selfupdate=<SECONDS>`: Configure an optional CRON job that auto-updates Juliaup if `<SECONDS>` has a value larger than 0. The actual value controls how often the CRON job will run to check for a new Juliaup version in seconds. The default value is 0, i.e. no CRON job will be created.
+- `--startup-selfupdate=<MINUTES>`: Configure how often Julia will check for new versions of Juliaup when Julia is started. The default is every 1440 minutes.
+- `-p=<PATH>` (or `--path`): Configure where the Julia and Juliaup binaries are installed. The default is `~/.juliaup`.
+
+## Alternative installation methods
+
+Note that we recommend the following methods _only_ if none of the installation methods described above work for your system.

```

Comment:
This whole document is catered towards personal use of Julia, which of course is often nice.

In university labs, though, admins often want to install Julia in a central location. Should we give a pointer here or somewhere about that? That would also cover people who don't like to pipe output of `curl` into `sh` for security reasons.

This could be an additional section at the end of this document. (I am not suggesting this idea should hold back this PR -- it definitely shouldn't! Just thinking out loud.

Reply:
Yes, lets handle that in a follow up PR! Or even better, fix Juliaup so that it works in those scenarios ;)